pkg/util/steps/runner.go

```diff
@@ -18,7 +18,12 @@ import (
 
 // FriendlyName returns a "friendly" stringified name of the given func.
 func FriendlyName(f interface{}) string {
-	return runtime.FuncForPC(reflect.ValueOf(f).Pointer()).Name()
+	return strings.TrimSuffix(runtime.FuncForPC(reflect.ValueOf(f).Pointer()).Name(), "-fm")
+}
+
+// FriendlyName returns a "friendly" stringified name of the given func.
+func ShortFriendlyName(f interface{}) string {
+	return strings.TrimPrefix(FriendlyName(f), "github.com/Azure/ARO-RP/")
 }
 
 func shortName(fullName string) string {
```

Comment:
I think while we're here also changing shortName to something a more descriptive might be helpful to understand the diff between `FriendlyName` and `shortName`. Something like `StripPath`